test/hotspot/jtreg/gc/stress/TestStressG1Humongous.java

```diff
@@ -50,11 +77,13 @@ public class TestStressG1Humongous{
     public static void main(String[] args) throws Exception {
         // Limit heap size on 32-bit platforms
         int heapSize = Platform.is32bit() ? 512 : 1024;
-        // Heap size, region size, threads, humongous size, timeout
-        run(heapSize, 4, 3, 1.1, 120);
-        run(heapSize, 16, 5, 2.1, 120);
-        run(heapSize, 32, 4, 0.6, 120);
-        run(heapSize, 1, 7, 0.6, 600);
+        // Region size, threads, humongous size, and timeout passed as @run arguments
+        int regionSize = Integer.parseInt(args[0]);
+        int threads = Integer.parseInt(args[1]);
+        double humongousSize = Double.parseDouble(args[2]);
+        int timeout = Integer.parseInt(args[3]);
+        
+        run(heapSize, regionSize, threads, humongousSize, timeout);
```

Comment:
I think you can remove the `run()` and inline the whole thing into `main()`. Just remove the lines in the middle.

Reply:
Made these changes. Thanks @shipilev